src/HunspellPHP/Hunspell.php

```diff
@@ -144,10 +156,13 @@ protected function stemCommand($input)
      */
     protected function preParse($input, $words)
     {
-        $result = explode(PHP_EOL, trim($input));
-        unset($result[0]);
-        $words = array_map('trim', explode(" ", $words));
+        $result = explode("\n", trim($input));
+        array_shift($result);
```

Comment:
I forgot to mention in the notes that I also changed the split char from " " to a regex \W because when people search for things on my site like blue/black it chokes up. I also added a catch in case something else breaks a warning is not thown if vars of two sizes are going to be passed to array_combine.